### PR TITLE
add uninstall instruction

### PR DESCRIPTION
**Pull request by linsun**

### CODE REVIEW

**Comment by linsun** [comment]
@lgadban a quick review?

**Comment by lgadban** [comment]
Can we add an additional note that says you may need to clear the `/root/.bumblebee` directory if builds we done as `root` or via `sudo`?

**Comment by linsun** [comment]
@lgadban added, PTAL!  thanks!

File: README.md [hunk 1]
```diff
@@ -64,6 +64,20 @@ sudo setcap cap_sys_resource,cap_sys_admin,cap_bpf+eip $(which bee)
...
 
+### Uninstall
+
+If you installed `bee` via the install script, you can uninstall `bee` by removing the directory:
+
+```bash
+rm -rf $HOME/.bumblebee/
+```
+
+If you built `bee` as root or via sudo, remove the following directory:
```
Comment (lgadban) [inline README.md:75]:
```suggestion
If you used `bee` as root or via sudo, remove the following directory:
```

**Review by lgadban — APPROVED** [review]

**Pull request merged**